radixdlt/babylon-wallet-android
# [ABW-3622] Select fee payer

[Jira ticket](https://radixdlt.atlassian.net/browse/ABW-3622)

## Description
This PR updates fee payer account selection UI.


## How to test

1. Launch a transfer transaction
2. On the transaction review screen tap "Customize" from Transaction Fee section
3. On the "Customize Fees" sheet tap "Change" from "Pay Fee From" section
4. Verify the account selection

## Screenshot
![Screenshot_20240729-151350](https://github.com/user-attachments/assets/3982f9ca-b8d0-4e68-acc6-8b27c09bf83e)

## PR submission checklist
